examples/datadog-sink/README.md

```diff
@@ -0,0 +1,56 @@
+# Datadog sink example
+
+The solution helps you set up a log-streaming pipeline from Stackdriver Logging to Datadog.
+
+## Instructions
+
+1. Fill the required variables in the `terraform.tfvars` file located in this directory.
+
+2. Verify the IAM roles for your Terraform service account:
+    - `roles/logging.configWriter` on the project (to create the logsink)
+    - `roles/iam.admin` on the project (to grant write permissions for logsink service account)
+    - `roles/serviceusage.admin` on the destination project (to enable destination API)
+    - `roles/pubsub.admin` on the destination project (to create a pub/sub topic)
+    - `roles/serviceAccount.admin` on the destination project (to create a service account for the logsink subscriber)
+
+2. Run the Terraform automation:
+    ```
+    terraform init
+    terraform apply
+    ```
+
+    You should see similar outputs as the following:
+
+    ![Screen Shot 2019-12-09 at 11.10.50 AM.png](https://github.com/smbreslow/terraform-google-log-export/raw/master/examples/datadog-sink/screenshots/Screen%20Shot%202019-12-09%20at%2011.40.44%20AM.png)
+
+3. In the GCP console, under `IAM > Service Accounts`, find the Datadog service account and create a set of JSON credentials:
+
+    ![Screen Shot 2019-12-09 at 11.10.22 AM.png](https://github.com/smbreslow/terraform-google-log-export/raw/master/examples/datadog-sink/screenshots/Screen%20Shot%202019-12-09%20at%2011.10.22%20AM.png)
```

Comment:
This can be done via Terraform as well (see the service_account_key resource).

Reply:
I don't think this is sufficient -- the service_account_key resource appears to only be able to export the public key (at least according to https://www.terraform.io/docs/providers/google/d/datasource_google_service_account_key.html#public_key), but Datadog needs the full JSON you get from the GUI or gcloud command.

Reply:
You can slo retrieve the private key json: https://www.terraform.io/docs/providers/google/r/google_service_account_key.html#private_key

Reply:
Thanks for pointing this out -- I've found the Terraform docs terribly difficult to navigate